app/javascript/components/supplemental_application/sections/ConfirmedHouseholdIncome.js

```diff
@@ -0,0 +1,122 @@
+import React from 'react'
+import { isNil } from 'lodash'
+
+import FormGrid from '~/components/molecules/FormGrid'
+import FormGroupTextValue from '~/components/atoms/FormGroupTextValue'
+import { RadioGroup, Radio, Text } from 'react-form';
+
+const YesNoRadioGroup = ({field}) => {
+  return (
+    <div className='radio-group-inline'>
+      <RadioGroup field={field}>
+        {(group) => (
+          <React.Fragment>
+            <p className="radio-inline">
+              <Radio group={group} value={`true`} id={`${field}-yes`}/>
+              <label className="radio-inline_label" htmlFor={`${field}-yes`}>Yes</label>
+            </p>
+            <p className="radio-inline">
+              <Radio group={group} value={`false`} id={`${field}-no`}/>
+              <label className="radio-inline_label" htmlFor={`${field}-no`}>No</label>
+            </p>
+          </React.Fragment>
+        )}
+      </RadioGroup>
+    </div>
+  )
+}
+
+const getAMIPercent = (income, ami) => {
+  console.log('Calculating the ami percent', income)
+  if (isNil(income))
+    return "Enter HH Income"
+  if (isNil(ami))
+    return "Error: AMI unknown"
+  return income / ami
+}
+
+
+
+class ConfirmedHouseholdIncome extends React.Component {
+
+  state = {
+    hudAMI: null,
+    stateAMI: null
+  }
+
+  componentDidMount() {
+    // Display a calculated annual income if applicant only provided monthly income
+    if (isNil(this.props.application.annual_income)) {
+      let annualIncome = this.props.application.monthly_income * 12
+      this.props.formApi.setValue('annual_income', annualIncome)
+    }
+
+    console.log('ConfirmedHouseholdIncome did mount')
+    // Fetch initial AMI values based on household size.
```

Comment:
I really do think that the AMI work should wait until another story, but here's one proposed way to do it using application state (here and in and componentDidUpdate()). Any initial thoughts on this as an idea?

AMI is just based on householdSize, not income

Reply:
I think we do not need `componentDidUpdate` and it can be handled in the `render()` ( like anual_income)